tests/testthat/test-key.R

```diff
@@ -1,3 +1,14 @@
+test_that("EIA key availability startup message works as expected", {
```

Comment:
I don't know if we need to test this one. We should not need to call a function like `readRenviron()`. Plus, that is not necessarily where a user will store their credentials. In fact it won't be the case in our CI/CD pipeline either. Definitely do not want to add `devtools` as any kind of package requirement. It brings in like a million other packages. I'd say just remove this test. If you want, we can include a covr file exclusion for zzz.R.